remo/base/content_urls.py

```diff
@@ -61,6 +61,36 @@
             model=FunctionalArea, form_class=FunctionalAreaForm,
             success_url=reverse_lazy('list_functional_areas')),
         name='edit_functional_area'),
+    url('^mobilising_skills/$',
+        BaseListView.as_view(
+            model=MobilisingSkill,
+            create_object_url=reverse_lazy('create_mobilising_skills')),
+        name='list_mobilising_skills'),
+    url('^mobilising_skills/new/$',
```

Comment:
Do mobilising `skills` and `interestes` need to be separate or can be combined into one entry?

Reply:
I think it's better to have it separate as we might introduce a new area, see if there is interest for it, and then start to train people on it